_i18n/es.yml

```diff
@@ -152,7 +152,7 @@ services-header:
    subtitle: Adaptadas a ti
 services-subheader:
    title: Siempre escuchando, en constante evolución 
-   subtitle: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent ac purus tortor. Praesent et laoreet lorem. Curabitur aliquet magna velit, vitae tincidunt magna laoreet et. Fusce cursus erat at nunc aliquet cursus. Praesent elit tortor, ornare vitae velit at, hendrerit dictum nunc. Nam cursus convallis enim vitae commodo. Nulla facilisi. Nulla quis felis vel risus dignissim sollicitudin ut ut lacus. Curabitur lacus ex, lacinia sed nunc luctus, dignissim tristique massa. Duis in purus sit amet nunc varius finibus id sed.
+   subtitle: Entendemos que cada uno es diferente, por lo que adaptamos nuestro enfoque a las necesidades de cada negocio y organización. Le ayudaremos a construir software bien hecho que sea relevante a sus necesidades. Proporcionamos <strong>equipos</strong> para crear software de calidad, que aporta valor, dentro de las fechas establecidas; <strong>consejo experto</strong> para acelerar sus iniciativas y <strong>formación y coaching</strong> para proveer a sus desarrolladores con las habilidades y actitud necesarias.
```

Comment:
Actually, on a second thought, maybe "dentro de los plazos establecidos" sounds a bit better in Spanish as a replacement for "in-time". What do you think?